### ISSUE TITLE
gpu_name doesn't show up on OpenGL apps

### ISSUE THREAD

**Issue by Cherenkov11** [flightlessmango/MangoHud#580]
Last version that worked correctly was v0.6.1, Vulkan apps shows `gpu_name` right in the latest version.

All following commands were executed with `MANGOHUD_CONFIG=full`

![Captura de pantalla de 2021-08-01 13-50-39](https://user-images.githubusercontent.com/29878131/127782677-ca9ab1e9-47fc-4af9-ab3e-5aa12888b20f.png)
(`glxgears` with mangohud v0.6.1 shows `gpu_name` correctly)

![Captura de pantalla de 2021-08-01 13-53-51](https://user-images.githubusercontent.com/29878131/127782681-f298f862-eb46-4ef2-8015-b671c6beeffc.png)
(`glxgears` with mangohud v0.6.5 shows no `gpu_name`)

![Captura de pantalla de 2021-08-01 14-10-49](https://user-images.githubusercontent.com/29878131/127782682-2260a32f-d4a7-4c03-9872-8a547d8b61bd.png)
(`vkcube` with mangohud v0.6.5 shows `gpu_name` correctly)

**Comment by fls2018** [comment]
I've got the same issue but not only with OpenGL apps but Proton games as well.

Here's a full hud from NMS:

![20210909201840_1](https://user-images.githubusercontent.com/43592670/132749373-042a0ac9-9f16-45c5-a149-ff7fe454b477.jpg)

vkcube works, unsure if any native Linux vulkan games work on steam.

**Comment by jackun** [comment]
Should be fixed in `develop` branch.

**Comment by Cherenkov11** [comment]
Yes, the _develop_ branch fix the issue. I will close this when d1617818cd77e002a64761dcf77ab1505d22ace6 reach the _master_ branch and a new release is deployed

![Captura de pantalla de 2021-09-09 21-06-25](https://user-images.githubusercontent.com/29878131/132787535-8a1ffc6c-36f2-43a8-9f8e-57a9ba24b6a4.png)
